target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-whw01v1.dts

```diff
@@ -0,0 +1,238 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "qcom-ipq4019-ap.dk01.1.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	model = "Linksys WHW01v1";
+	compatible = "linksys,whw01v1";
```

Comment:
If v1 stands for "version 1", please separate it by a space for the model and a hyphen for the compatible.

This will require corresponding adjustments to DTS file name and device definition etc.